### ISSUE TITLE
Find in files crashes when launched in Insiders build

### ISSUE THREAD

**Issue by IllusionMH** [microsoft/vscode#84250]
Issue Type: <b>Bug</b>

**Workaround:** set `search.location` to `"sidebar"` and reload window.

In latest Insiders build I'm unable to use Find in Files. Search panel is just empty.

Next error in console:

```
ERR Cannot read property 'searchInput' of undefined: TypeError: Cannot read property 'searchInput' of undefined
    at e.hasSearchPattern (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:6071:109)
    at O.get enabled [as enabled] (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:5354:665)
    at f.updateEnabled (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:934:329)
    at f.updateClass (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:934:232)
    at f.render (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:933:425)
    at file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:938:889
    at Array.forEach (<anonymous>)
    at S.push (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:938:478)
    at file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:1020:548
    at Array.forEach (<anonymous>)
    at file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:1020:521
    at e.setActions (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:5061:489)
    at e.renderHeader (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:5060:796)
    at e.render (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:537:6)
    at e.render (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:5060:69)
    at g.create (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:6087:385)
    at e.showComposite (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:5435:390)
    at e.doOpenComposite (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:5434:252)
    at e.openComposite (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:5433:788)
    at e.openPanel (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:5629:642)
    at B.restoreWorkbench (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:5668:774)
    at file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:5664:467
    at h.invokeFunction (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:1532:149)
    at B.startup (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:5664:94)
    at Q.open (file:///<USER DIR>/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:6212:929)
```

VS Code version: Code - Insiders 1.41.0-insider (fdeb9aa9b8306a2d3da989ccb9d2ad288d0ae347, 2019-11-08T05:28:30.394Z)
OS version: Windows_NT x64 10.0.18362

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (12 x 3192)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.72GB (15.77GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>Extensions: none

**Comment by Dayjo** [comment]
This has become an issue for me too;
```
Version: 1.41.0-insider
Commit: fdeb9aa9b8306a2d3da989ccb9d2ad288d0ae347
Date: 2019-11-08T05:26:43.949Z
Electron: 6.1.4
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 19.0.0
```

The 'workaround' "works"

**Comment by JacksonKearl** [comment]
\duplicate #84220

**Comment by JacksonKearl** [comment]
Thanks for the stack trace! This is a duplicate of #84220 and closed in 777ad66, the next insiders will have the fix.
